internal/osbuild2/ostree_deploy_stage.go

```diff
@@ -0,0 +1,29 @@
+package osbuild2
+
+// Options for the org.osbuild.ostree.deploy stage.
+type OSTreeDeployStageOptions struct {
+
+	OsName string `json:"osname"`
+
+	Ref string `json:"ref"`
+
+	Mounts []string `json:"mounts"`
+
+	Rootfs Rootfs `json:"rootfs"`
+
+	KernelOpts []string `json:"kernel_opts"`
```

Comment:
`Mounts` and `KernelOpts` should be marked `omitempty`.  It probably doesn't make a difference functionally but it can sometimes so we'd rather mirror the stage schema as accurately as possible for this.

(I'm sure there are a lot of cases where we don't do this though)

Reply:
fixed!